docs/usage/shoot_high_availability.md

```diff
@@ -49,4 +49,16 @@ If you already have a shoot cluster with non-HA control plane then following upg
 
 **Disallowed Transitions**
 
-If you have already set-up a HA shoot control plane with `node` failure tolerance then an upgrade to `zone` failure tolerance is currently not supported, mainly because already existing volumes are bound to the zone they were created in.
\ No newline at end of file
+If you have already set-up a HA shoot control plane with `node` failure tolerance then an upgrade to `zone` failure tolerance is currently not supported, mainly because already existing volumes are bound to the zone they were created in.
```

Comment:
```suggestion
If you have already set-up an HA shoot control plane with `node` failure tolerance, then an upgrade to `zone` failure tolerance is currently not supported, mainly because already existing volumes are bound to the zone they were created in originally.
```

Reply:
I really have no idea why this is shown as a change but I will apply your suggestion 😄